modules/analyzing-the-collector-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: CONCEPT
+[id="analyzing-the-collector-logs_{context}"]
+
+= Analyzing the Collector logs
+
+The first thing you should look at are the logs from any failing collectors. Having access to your own logs is extremely valuable. Depending on your environment and access, you can obtain these logs in three ways:
```

Comment:
```suggestion
The first thing you should examine are the logs from any failing collectors. Having access to your own logs is extremely valuable. Depending on your environment and access, you can obtain these logs in three ways:
```

**issue:** Style guide recommendation.

Reply:
Also I would rewrite the first sentence as "First, you should examine the logs from failing collectors." Its more direct. **issue:** Then the next line "Having access ... " can be removed. Its just stating something and doesn't adds any value to the process. **nitpick:** For the third sentence, I would add `access rights`, or `access permissions`.

Reply:
ack